.travis.yml

```diff
@@ -14,12 +14,21 @@ script:
   - npm test
   - npm run coverage
 
-addons:
-  firefox: 'latest'
 
 before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 
 after_success:
   - npm run coverage-publish
+
+env:
+  - CXX=g++-4.8
```

Comment:
None that we really need, travis was just complaining and I wanted to fix that (those native modules are mostly used through aegir)